pkg/cloudprovider/backup_cache.go

```diff
@@ -41,15 +41,17 @@ type backupCache struct {
 	// This doesn't really need to be a map right now, but if we ever move to supporting multiple
 	// buckets, this will be ready for it.
 	buckets map[string]*backupCacheBucket
+	logger  *logrus.Logger
 }
 
 var _ BackupGetter = &backupCache{}
 
 // NewBackupCache returns a new backup cache that refreshes from delegate every resyncPeriod.
-func NewBackupCache(ctx context.Context, delegate BackupGetter, resyncPeriod time.Duration) BackupGetter {
+func NewBackupCache(ctx context.Context, delegate BackupGetter, resyncPeriod time.Duration, logger *logrus.Logger) BackupGetter {
 	c := &backupCache{
 		delegate: delegate,
 		buckets:  make(map[string]*backupCacheBucket),
+		logger:   logger,
```

Comment:
suggestion, which you may not agree with. `backupCache` is unexported, and being returned inside the `BackupGetter` interface so if you did something like this
```
type backupCache struct {
      *logrus.Logger
      ...
}
```
Then anywhere you call `c.logger.Info` you can use `c.Info`.